### PR TITLE
Added call to compute_initial_accel

### PR DESCRIPTION
**Pull request by cahaynes**

### CODE REVIEW

File: src/solver/src/newmark_solver.C [hunk 1]
```diff
@@ -65,6 +65,12 @@ namespace GRINS
 
     context.system->deltat = this->_deltat;
 
+    libMesh::NewmarkSolver * newmark = NULL;
+
+    newmark = libMesh::cast_ptr<libMesh::NewmarkSolver *>(context.system->time_solver.get());
```
Comment (pbauman) [inline src/solver/src/newmark_solver.C]: Collapse this with the line above please.

**Comment by cahaynes** [comment]
Hopefully that commit message actually make sense.

**Comment by pbauman** [comment]
Cool. Could you please squash those commits together and then I'll merge.

**Comment by pbauman** [comment]
Thanks!

**Pull request merged**